app/vmauth/main.go

```diff
@@ -103,11 +99,12 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 	for _, h := range headers {
 		r.Header.Set(h.Name, h.Value)
 	}
-	proxyRequest(w, r)
+	catchPanic()
```

Comment:
catchPanic will work incorrectly - it will be finished before actual panic is happen. 

 Solution - move panic catch code out of function and use it in place

Reply:
I have removed this function and it catch in prev place
```golang
func proxyRequest(w http.ResponseWriter, r *http.Request) {
	defer func() {
		err := recover()
		if err == nil || err == http.ErrAbortHandler {
			// Suppress http.ErrAbortHandler panic.
			// See https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1353
			return
		}
		// Forward other panics to the caller.
		panic(err)
	}()
	getReverseProxy().ServeHTTP(w, r)
}	
```